toolkit/scripts/check_entangled_specs.py

```diff
@@ -49,10 +49,6 @@
         "SPECS/python-flit/python-flit.spec",
         "SPECS/python-flit-core/python-flit-core.spec"
     ]),
-    frozenset([
```

Comment:
Double checking that removing this is the right thing to do?  I think you’re ok, just looking for clarity on why this can be removed now.

Reply:
You are right, I have read more about it and it seems like it is recommended to have the same version of `influx-cli` as the `influxdb` or some queries might not work. I put this back and synced the versions. Unfortunately, the `influx-cli` does not have version 2.7.5 yet